packages/fe-container/src/hooks/useAppData/useAppData.ts

```diff
@@ -39,7 +39,7 @@ const notHref = (plugin: InternalPlugin) => plugin.integrationMode && plugin.int
 
 const registerPlugins = (configuration: Configuration, user: Partial<User>) => {
   [...configuration.plugins || [], ...configuration.internalPlugins || []].sort(pluginsSorter).forEach(registerPlugin)
-  finish(user)
+  finish(user, configuration.shared || {})
```

Comment:
instead of an or here, you should use the default argument in the function definition